src/types/types.ts

```diff
@@ -17,18 +17,17 @@ export enum QuestionType {
 }
 
 export enum AnswerType {
-  LField = 'lField',
-  SField = 'sField',
+  LargeInput = 'largeInput',
+  SmallInput = 'smallInput',
   Dropdown = 'dropdown',
-  Radio = 'radio',
   Calendar = 'calendar',
 }
 
 export type Client = {
   id: string;
   fullName: string;
   createdAt: Date;
-  answers?: Map<string, Map<string, string>>;
+  answers: Map<string, Map<string, any>>;
```

Comment:
Should answers be optional for users that have yet to fill out an intake form? (like when they just created an account)

Reply:
no, it should be mandatory, but can be empty if they havent filled an intake form out